bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Tab = ({
+  index,
+  icon,
+  activeKey,
+  tabArea,
+  onClickEvent,
+}) => {
+  let tabClass = 'w-full';
+
+  if (index === activeKey) {
+    tabClass += 'bg-gray-100 border-l-8 border-blue-600';
+  } else {
+    tabClass += 'border-b-2 border-gray-200';
+  }
+
+  return (
+    <li className={tabClass} onClick={onClickEvent} aria-hidden="true">
+      <a href={tabArea} className="p-8 block justify-center flex">
+        <img src={`images/${icon}.svg`} className="fill-current" alt="" />
```

Comment:
i think instead of adding img to each component, let create `Icon` component which receives only one props may be `icon` and replace it everywhere?

Reply:
done